TypeCobol/Compiler/CodeElements/Expressions/StorageArea.cs

```diff
@@ -520,8 +521,8 @@ public IntrinsicFunctionCall(ExternalName intrinsicFunctionName, CallSiteParamet
 		}
 
 		public ExternalName IntrinsicFunctionName { get; private set; }
-		public override string FunctionName { get { return IntrinsicFunctionName.Name; } }
-		public override Token FunctionNameToken { get { return IntrinsicFunctionName.NameLiteral.Token; } }
+		public override string FunctionName { get { return IntrinsicFunctionName?.Name; } }
```

Comment:
Done in 89bb05784e8d5fe7c3ebefa92bdadf38aa45ffa4. Missing attributes have been added, note that it is not necessary to repeat them on overriding methods/properties because these are implicitly inherited.